tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1145,56 +1145,20 @@ generic_config_updater/test_pg_headroom_update.py:
 #######################################
 #####           gnmi              #####
 #######################################
-gnmi/test_gnmi.py:
-  skip:
```

Comment:
There is another build image PR need merge: 
https://github.com/sonic-net/sonic-buildimage/pull/21849

If this PR merge first, then that PR may block by test case